src/api/hooks/video.ts

```diff
@@ -109,3 +112,50 @@ export const useMostViewedVideos = (variables?: GetMostViewedVideosQueryVariable
     ...rest,
   }
 }
+
+type MostViewedVideosAllTimeOpts = QueryHookOptions<GetMostViewedVideosAllTimeQuery>
+export const useMostViewedVideosAllTimeIds = (
+  variables?: GetMostViewedVideosAllTimeQueryVariables,
+  opts?: MostViewedVideosAllTimeOpts
+) => {
+  const { data, ...rest } = useGetMostViewedVideosAllTimeQuery({ ...opts, variables })
+  return {
+    mostViewedVideosAllTime: data?.mostViewedVideosAllTime,
+    ...rest,
+  }
+}
+
+export const useMostViewedVideosAllTime = (
+  variables?: GetMostViewedVideosAllTimeQueryVariables,
+  opts?: MostViewedVideosAllTimeOpts
+) => {
+  const { mostViewedVideosAllTime } = useMostViewedVideosAllTimeIds(variables, opts)
+
+  const mostViewedVideosAllTimeIds = useMemo(() => {
+    if (mostViewedVideosAllTime) {
+      return mostViewedVideosAllTime.map((item) => item.id)
+    }
+    return null
+  }, [mostViewedVideosAllTime])
```

Comment:
`useMemo` likely doesn't make much sense here, the performance cost of memoization will likely be higher than the computation being done